rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -294,31 +281,6 @@ def _to_binary_inst(msg):
         return msg
 
 
-def _to_time_inst(msg, rostype, inst=None):
-    # Create an instance if we haven't been provided with one
-
-    if rostype == "time" and msg == "now":
-        return ROSClock().now().to_msg()
```

Comment:
Looks like this is removing a feature where the string `"now"` will be translated to the current time. I guess this feature was not documented, so it's ok to remove? Is that what you're thinking too?

Reply:
Ah, I didn't care about that, sorry. Will confirm the reason why this was added!

Reply:
Introduced here: 8e3eb968369ebf63422139f18657239e006495cd